### PR TITLE
detect Perl code as Perl instead of Raku

### LINKED ISSUE #523: GitHub: Some *.pl files are being detected as Raku
If you "Click for language details" on the colorful bar on the repo's home page, our files are currently shown as C++ 99.9%, CMake 0.1%, Perl 0.0%, PowerShell 0.0%, Raku 0.0%, Batchfile 0.0%.

Raku is the [new name](https://en.wikipedia.org/wiki/Raku_\(programming_language\)) for Perl 6.

The files in question, [`tests/std/tests/Dev09_056375_locale_cleanup/custombuild.pl`](https://github.com/microsoft/STL/blob/482f1d80886a7176674ec664569264b6bbbc4cfd/tests/std/tests/Dev09_056375_locale_cleanup/custombuild.pl) and [`tests/std/tests/VSO_0000000_any_calling_conventions/custombuild.pl`](https://github.com/microsoft/STL/blob/482f1d80886a7176674ec664569264b6bbbc4cfd/tests/std/tests/VSO_0000000_any_calling_conventions/custombuild.pl), are part of our legacy test harness and they are definitely Perl, not Raku.

While this is an extremely minor cosmetic issue, it can be fixed. GitHub's "linguist" tool is responsible for this language detection, and it supports [explicit language overrides via `.gitattributes`](https://github.com/github/linguist#using-gitattributes).

We previously fixed issue #1 this way:

https://github.com/microsoft/STL/blob/482f1d80886a7176674ec664569264b6bbbc4cfd/.gitattributes#L7-L9

We should make a similar change to mark all `*.pl` files as Perl.

### PR DESCRIPTION
**Pull request by Svido**
Fixes #523

# Description
Modify .gitattributes to detect Perl code as Perl instead of Raku.
Verified by local installation of github-linguist.

# Checklist

Be sure you've read README.md and understand the scope of this repo.

If you're unsure about a box, leave it unchecked. A maintainer will help you.

- [x] Identifiers in product code changes are properly `_Ugly` as per
  https://eel.is/c++draft/lex.name#3.1 or there are no product code changes.
- [ ] The STL builds successfully and all tests have passed (must be manually
  verified by an STL maintainer before automated testing is enabled on GitHub,
  leave this unchecked for initial submission).
- [x] These changes introduce no known ABI breaks (adding members, renaming
  members, adding virtual functions, changing whether a type is an aggregate
  or trivially copyable, etc.).
- [x] These changes were written from scratch using only this repository,
  the C++ Working Draft (including any cited standards), other WG21 papers
  (excluding reference implementations outside of proposed standard wording),
  and LWG issues as reference material. If they were derived from a project
  that's already listed in NOTICE.txt, that's fine, but please mention it.
  If they were derived from any other project (including Boost and libc++,
  which are not yet listed in NOTICE.txt), you *must* mention it here,
  so we can determine whether the license is compatible and what else needs
  to be done.

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -7,3 +7,7 @@
 # Ensure GitHub detects our C++ code as C++ code.
 /stl/inc/** linguist-language=C++
 /stl/src/** linguist-language=C++
+
+# Ensure Github detects out Perl legacy test harness code as Perl code instead of Raku
```
Comment (BillyONeal) [inline .gitattributes]: It seems like all .pls should be perl?

Comment (CaseyCarter) [inline .gitattributes]:
```suggestion
# Ensure Github detects our Perl legacy test harness code as Perl code instead of Raku
```

**Comment by BillyONeal** [comment]
Thanks for your contribution!

**Comment by StephanTLavavej** [comment]
Yay! 😸

**Pull request merged**